### ISSUE TITLE
[Console] QuestionHelper::getHiddenResponse() not working with space in project directory name

### ISSUE THREAD

**Issue by Yendric** [symfony/symfony#40190]
(This is my first ever GitHub issue, so let me know if I'm doing anything wrong!)

**Symfony version(s) affected**: symfony/console ^5.2.3 (latest)

**Description**  
When using the getHiddenResponse method of the QuestionHelper class (which is called when running askQuestion with a hidden question) in a project of which the location has a space in it, it will error out and try to execute the first part of the project location, eg.:
My project is located in 'D:\My Projects\My Cool Project', it will try to execute 'D:\My'.

**How to reproduce**  
Create a php application with the Symfony console component inside of a folder with a space in the name (see example above).  Then run askQuestion with a Question object that has hidden set to true.

**Possible Solution**  
Changing this line: `$exe = __DIR__.'/../Resources/bin/hiddeninput.exe';` to this: `$exe = '"' . __DIR__ . '/../Resources/bin/hiddeninput.exe' . '"';` in Helper\QuestionHelper.php line 405.
This should fix the problem stated above.

**Additional context**  
I found this 'bug' in Laravel 8 on Windows whilst trying to create a console command with a secret input.

**Comment by Nyholm** [comment]
Thank you for this issue. 

When running a console application, arguments are separated with a space. You can be explicit/clear that it is indeed one string by quoting it. 

```
# This does not work
> What is your project directory?
D:\My Projects\My Cool Project

# This will work
> What is your project directory?
"D:\My Projects\My Cool Project"
```

On a unix system you can also "protect" the space with a backslash. 

```
# This does not work
> What is your project directory?
/home/user/My Projects/My Cool Project

# This will work
> What is your project directory?
/home/user/My\ Projects/My\ Cool\ Project
```

----------

A side note, I found it much easier configurating web servers etc if the project path is space free.

**Comment by Yendric** [comment]
Im not giving a directory as an argument. It actually just crashes the console application when trying to ask a hidden question, even before you can answer it. 
Thanks for the webserver advice.
I have fixed the issue in #40192

**Comment by Nyholm** [comment]
Sorry. I probably misunderstand you then. I'll follow up on the PR.
